spec/features/user_management_spec.rb

```diff
@@ -1,9 +1,13 @@
 require 'rails_helper'
 
-def sign_up(email: 'barney@barney.com',
+def sign_up(first_name: 'Barney',
```

Comment:
I'd like to see more tests here - how are you handling users with only 1 name? What happens if a user only fills one field, or no fields?